core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -230,23 +230,30 @@ THE SOFTWARE.
     </div>
 
     <j:if test="${layoutType!='full-screen'}">
-    <footer>
+    <footer class="page-footer">
       <div class="container-fluid">
-        <div class="row">
-          <div class="col-md-6" id="footer"></div>
-          <div class="col-md-18">
-            <span class="page_generated">${%Page generated}: <i:formatDate value="${h.getCurrentTime()}" type="both" dateStyle="medium" timeStyle="long"/></span>
-            <j:if test="${!empty(it.api)}">
-               <span class="rest_api"><a href="api/">REST API</a></span>
-            </j:if>
-            <span class="jenkins_ver"><a href="${h.getFooterURL()}">Jenkins ver. ${h.version}</a></span>
-            <j:if test="${extensionsAvailable}">
-            <j:forEach var="pd" items="${h.pageDecorators}">
-              <st:include it="${pd}" page="footer.jelly" optional="true" />
-            </j:forEach>
-            </j:if>
+        <div class="page-footer__flex-row">
+          <div class="page-footer__footer-id-placeholder" id="footer"></div>
+
+          <j:if test="${!empty(it.api)}">
+            <div class="page-footer__links rest_api hidden-xs">
+              <a href="api/">
+                REST API<l:svgIcon href="${resURL}/images/material-icons/svg-sprite-navigation-symbol.svg#ic_arrow_forward_24px" />
```

Comment:
I know it's not related per se with the changes, but maybe it's a good momento to think about making these external links so that they open in another tab?

Reply:
You got a point, I'll update the version link to make it open in another tab. The api link is internal so I'll leave it be.